src/Page/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import { ThemeProvider } from '@emotion/react';
+import { StyledHeader } from './pageStyles';
+import theme from 'src/styles/theme';
+
+ /**
+ * Header Component
+ * 
+ * The Header component is used to render the header.
+ */
+const Header: FC = props =>
+{
+    const { children } = props;
```

Comment:
Also, please destructure  `children` in the function call as opposed to in the body of the function as we do in other components.

Reply:
done